src/storage/pubsub/topic.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "index.h"
+
+#include <cc_bstring.h>
+
+/*
+ * a topic is an endpoint that clients can subscribe to, equivalent to
+ * "channel" in the original redis protocol.
```

Comment:
is there any reason to call it topic instead of channel? do we refer to something else as channel? you could add the reason to the comment here